docs-java/connecting-to-ai-core.mdx

```diff
@@ -56,11 +63,11 @@ Here are three examples of how this can be done:
 1.  <details>
     <summary>Using a `.env` file</summary>
 
-    Create a `.env` file in the **working directory** from which you run your code.
+    Create a `.env` file (without specifying a filename) in the **working directory** from which you run your code.
     Add the following line:
 
     ```txt
-    AICORE_SERVICE_KEY={ "clientid": "...", "clientsecret": "...", "url": "...", "serviceurls": { "AI_API_URL": "..." } }
+    AICORE_SERVICE_KEY={...Fill it with JSON object copied above...}
```

Comment:
We could replace this whole section with a screenshot of the file (censored). It would explain the directory to place it in, the name and the fact that it has to be in one line instead of this entire paragraph.